### PR TITLE
App migration FAQ

### PR DESCRIPTION
**Pull request by shairyar**
This PR adds the instructions on how to migrate an app from one organization to another.

### CODE REVIEW

**Comment by jeffkreeftmeijer** [comment]
Hadn't thought about moving this to the public documentation, but I don't have any objections. I've asked for more reviews on this, so we can discuss.

**Comment by shairyar** [comment]
Sure, one of the customer who contacted us recently about the migration said he could not find any information around it in the docs, so i thought if might be a good idea to add.

**Review by thijsc — APPROVED** [review]

File: source/faq.html.md [hunk 1]
```diff
@@ -61,3 +63,19 @@ Also check our [known issues](/support/known-issues.html) page for issues that m
 ### How to enable two-factor authentication (2FA) for the AppSignal app?
 
 Please see our [two-factor authentication](/user-account/two-factor-authentication.html) page for more information.
+
+## Application
```
Comment (tombruijn) [inline source/faq.html.md:67]: I'd probably move this to the applications section, e.g. https://docs.appsignal.com/application/. The FAQ is meant to link to other deeper pages in the docs, not have original content on its own.

For now I'd put it on the overview page for applications under/above "remove an application".

**Review by xadamy — APPROVED** [review]

File: source/faq.html.md (hunk 1 above)
Comment (shairyar) [reply source/faq.html.md:67]: That is a good idea.

**Review by jeffkreeftmeijer — APPROVED** [review]
Approved once https://github.com/appsignal/appsignal-docs/pull/356#discussion_r490168343 is resolved.

**Review by tombruijn — APPROVED** [review]

**Pull request merged**